docs/components/fieldset/index.md

```diff
@@ -4,20 +4,56 @@ Fieldset is a component to aid in creating form components that require an under
 
 ## Label
 
-Provide a string to `@label` to render the text into the `<legend>` of the Fieldset. This is required.
+Required.
+
+Use either `@label` or `:label`.
+
+Provide a string to `@label` or components to `:label` to render the text into the `<legend>` of the Fieldset.
```

Comment:
Same comments here (and all the other files!) as above, but this is looking good!